src/api/module/murojaah/entity/index.ts

```diff
@@ -1,10 +1,23 @@
+export type History = {
+  id?: number;
+  murojaahType: number;
+  juz?: number;
+  surah?: number;
+  surahName?: string;
+  start?: number;
+  end?: number;
+  murojaahMethod: string;
```

Comment:
can we define all the methods here? `'xxx' | 'yyy'`

Reply:
did you mean the methods from `service` and `repository` folder?

if yes then currently we utilize the [Clean Architecture](https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html) methodology by separating business rule, logic, and db layer interaction

Reply:
by methods i mean murojaah methods

Reply:
ah, currently it hardcoded, later user should be able to create it's own murojaah method, also maybe we will provide some predefined murojaah method

the hardcoded todo note are defined here
https://github.com/RR-Ardhika/murojaah/pull/8/files#diff-4e975d29255ca390bc07a1c77aeaaec167d6aafdecd04bd46135440fe0341bccR15